docs/tutorials/fundamentals/installation.md

```diff
@@ -41,6 +39,12 @@ You may also want:
 - an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
 [venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
 
+```{note}
+New to Python or not familiar with virtual environments? You can use our
```

Comment:
```suggestion
New to Python or uncertain about conda, pip, and virtual environments? You can use our
```

Totally optional, but trying to make it more explicit in terms of terminology people could have heard about and we use below.